### DISCUSSION TITLE
Proposed Medical Changes to Address #2746

### DISCUSSION THREAD

**Discussion by bloxgate** [HaloSpaceStation/HaloSpaceStation13#2884] [Ideas]
As mentioned in #2746, necrosis is currently happening faster than it should and was a major problem even before then. As a workaround I'm going to propose that we make Peridaxon capable of healing necrotic organs if their damage has been healed to below the bruising point.

We could then add a small dose of peridaxon to the tri-addrenaline syringes. I'd also like some input on replacing the peridaxon autoinjectors (called tissue necrosis inhibitors in the kit) in the field medical kit with low dose tri-addrenaline autoinjectors as well.

Thoughts? Comments? Suggestions?

**Comment by Sgtwok** [comment]
Well making peridaxon actually useful would be a start.

**Comment by bloxgate** [comment]
There is a partial mitigation for this planned in #2959
